src/js/views/patients/shared/list_views.js

```diff
@@ -48,9 +47,10 @@ const CountView = View.extend({
     const filteredCollection = this.getOption('filteredCollection');
 
     return {
-      maximumCount: MAXIMUM_LIST_COUNT,
+      maximumCount: this.collection.length,
```

Comment:
Technically, this will be incorrect when the list hasn't reached the maximum. But it won't be used in those instances in any hbs template. Only when the list has reached the maximum.